esphome/components/font/__init__.py

```diff
@@ -109,9 +110,42 @@ def _compute_gfonts_local_path(value) -> Path:
     return _compute_local_font_dir(name) / "font.ttf"
 
 
+def get_filename_from_url(url):
+    # Regular expression pattern to match the file name at the end of the URL
+    pattern = r"/([^/]+)$"
+    match = re.search(pattern, url)
+    if match:
+        file_name = match.group(1)
```

Comment:
You might want to rename to filename for consistency with the method name
```suggestion
        filename = match.group(1)
```

Reply:
i renamed the method instead because file_name was used elsewhere already